src/Relation/Binary/Properties/Setoid.agda

```diff
@@ -0,0 +1,36 @@
+------------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some theory for Setoid
+------------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+
+open import Relation.Binary using (Rel; Symmetric; _Respects_; _Respects2_;
+                                                               Setoid)
+
+
+module Relation.Binary.Properties.Setoid {ℓ ℓ=} (A : Setoid ℓ ℓ=) where
+
+open import Function         using (_∘_)
+open import Relation.Nullary using (¬_)
+
+open Setoid A using (Carrier; _≈_; refl; sym; trans)
+
+_≉_ :  Rel Carrier _
+_≉_ x =  ¬_ ∘ (x ≈_)
+
+≉sym :  Symmetric _≉_
```

Comment:
Please add a dash between the operator and the name (e.g. `≉-sym`) as is standard.

Reply:
Done.